.github/workflows/create-parity-issue.yml

```diff
@@ -0,0 +1,41 @@
+name: create-parity-issue.yml
+
+on:
+  workflow_dispatch:
+    inputs:
+      prDescription:
+        description: PR description
+        default: 'No description provided'
+        required: true
+      prNumber:
+        description: PR number
+        required: true
+      prTitle:
+        description: PR title
+        required: true
+      sourceRepo:
+        description: repository PR is sourced from
+        required: true
+
+jobs:
+  createIssue:
+    name: create issue
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: joshgummersall/create-issue@main
```

Comment:
@gabog this action might be useful in writing a workflow to automatically create child issues from a parent botframework-sdk issue.